rosys/vision/detector_hardware.py

```diff
@@ -198,7 +198,7 @@ async def fetch_detector_info(self) -> DetectorInfo:
             raise DetectorException('Invalid response from detector')
 
         try:
-            model_info_dict = response.get('model_info', {})
+            model_info_dict = response.get('model_info', {}) or {}
```

Comment:
Tiny simplification:
```suggestion
            model_info_dict = response.get('model_info') or {}
```